src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -3884,17 +3884,14 @@ private static RelocType GetRelocType(TargetArchitecture targetArchitecture, ush
                             return 0;
                     }
                 }
-                case TargetArchitecture.Riscv64:
+                case TargetArchitecture.RiscV64:
                 {
                     const ushort IMAGE_REL_RISCV64_PC = 3;
-                    const ushort IMAGE_REL_RISCV64_JALR = 4;
 
                     switch (fRelocType)
                     {
                         case IMAGE_REL_RISCV64_PC:
                             return RelocType.IMAGE_REL_BASED_RISCV64_PC;
-                        case IMAGE_REL_RISCV64_JALR:
-                            return RelocType.IMAGE_REL_BASED_RISCV64_JALR;
```

Comment:
IMO, these code be updated or removed as well. Please check.
https://github.com/dotnet/runtime/blob/0cf461b302f58c7add3f6dc405873fb2212b513f/src/coreclr/jit/emitriscv64.cpp#L1546
https://github.com/dotnet/runtime/blob/0cf461b302f58c7add3f6dc405873fb2212b513f/src/coreclr/pal/inc/rt/ntimage.h#L1028

Reply:
I missed the comment. Please update in the next PR. Thank you.